### PR TITLE
[TR#27] Revised Tokens

### PR DESCRIPTION
**Pull request by Jeremy-Walton**
## Task

[TR #27](https://trello.com/c/wOBuyEcZ)

## Why?

After recent design efforts and a complete Figma library, we want to ensure all our base tokens match what Figma is using.

## What Changed

- [X] Update Input Height tokens
- [X] Refactor transition tokens to include all transition properties instead of just duration
- [X] Remove navigation transition token as it was not being used
- [X] Move flash transition into flash animation token
- [X] Add new Radius stop (12px)
- [X] Add sanity check npm command to run through checks and clean up after build commands

## Sanity Check

- [X] Have you updated any usage of changed tokens?
- [X] Have you updated the docs with any component changes?
- [X] Have you run linters?
- [X] Have you run prettier?
- [X] Have you tried building the css?
- [X] Have you tried building storybook?
- ~~Do you need to update the package version?~~

## Screenshots

Screenshot 2023-03-08 at 12 09 46 PM

### CODE REVIEW

**Comment by Jeremy-Walton** [comment]
[Revise Base Style Tokens](https://trello.com/c/wOBuyEcZ/27-revise-base-style-tokens)

**Comment by scriswell** [comment]
@Jeremy-Walton The screenshot shows the documentation for height but it's displaying as width.

File: src/core/tokens/base_tokens.scss [hunk 1]
```diff
@@ -250,10 +250,10 @@
...
   * @presenter Spacing
   */
-  --op-input-height-small: 2.4rem; // 24px
-  --op-input-height-medium: 3.2rem; // 32px
-  --op-input-height-large: 4.4rem; // 44px
-  --op-input-height-x-large: 5.6rem; // 56px
+  --op-input-height-small: 2.8rem; // 28px
+  --op-input-height-medium: 3.6rem; // 36px
+  --op-input-height-large: 4rem; // 40px
+  --op-input-height-x-large: 8.4rem; // 84px
```
Comment (scriswell) [inline src/core/tokens/base_tokens.scss:256]: @Jeremy-Walton Is this one being used as a default for `textarea`?

**Comment by scriswell** [comment]
@Jeremy-Walton I'm going to move the following into a follow-up PR so I can add a focus pattern for error buttons as well. I also want to fine-tune the CSS a bit to match Figma.
```
/**
  * @tokens Input Focus
  * @presenter Shadow
  */
  --rm-input-outer-focus: 0 0 0 var(--rm-border-width-large) var(--rm-color-primary-plus-four);
  --rm-input-inner-focus: inset var(--rm-border-all) var(--rm-color-primary-minus-three);
  --rm-input-focus: var(--rm-input-inner-focus), var(--rm-input-outer-focus);
}```

**Review by scriswell — APPROVED** [review]
@Jeremy-Walton Approved. I did have one comment relating to `--op-input-height-x-large` and also made a note for us to build a follow-up PR to track further enhancements to our button focus state. I created a card for that work as well.

File: src/core/tokens/base_tokens.scss (hunk 1 above)
Comment (Jeremy-Walton) [reply src/core/tokens/base_tokens.scss:256]: Not currently but that can be addressed in https://trello.com/c/Z3ThAj5l It's not currently used at all

**Comment by Jeremy-Walton** [comment]
@scriswell That's because the plugin I showed you doesn't have a Height Presenter. It only has a "Spacing" presenter which shows as a width.

**Comment by scriswell** [comment]
@Jeremy-Walton We may want to remove it then as it's a bit confusing and doesn't accurately document the effect of the custom property.

**Review by scriswell — APPROVED** [review]

**Pull request merged**